lib/themes/dosomething/paraneue_dosomething/js/components/CampaignSlide.js

```diff
@@ -35,20 +36,22 @@ class CampaignSlide extends React.Component {
           <div className="wrapper">
             {this.renderHeader()}
 
-            {this.campaigns.map(campaign => {
-              // Fix for local files / data missing by preventing null error
-              const image_uri = campaign.cover_image.default != null ? campaign.cover_image.default.sizes.landscape.uri : '';
+            <div className="container__cards">
```

Comment:
Switched to triad gallery in this commit: https://github.com/DoSomething/phoenix/pull/7058/commits/ab7261183167a782c8247b35455c04a67265a3c0

Heres what it looks like
<img width="1162" alt="screen shot 2016-09-20 at 10 22 53 am" src="https://cloud.githubusercontent.com/assets/897368/18674163/3f4e8f96-7f1c-11e6-828c-5313592887d1.png">
<img width="477" alt="screen shot 2016-09-20 at 10 23 06 am" src="https://cloud.githubusercontent.com/assets/897368/18674164/3f533852-7f1c-11e6-97ad-525394ea4c66.png">